share/spack/gitlab/cloud_pipelines/stacks/ml-darwin-aarch64/spack.yaml

```
@@ -0,0 +1,92 @@
+spack:
+  view: false
+
+  packages:
+    all:
+      require:
+      - target=aarch64
+
+  definitions:
+    - packages:
+        # Horovod
+        - py-horovod
+
+        # Hugging Face
+        - py-transformers
+
+        # JAX
+        - py-jax
+        - py-jaxlib
+
+        # Keras
+        - py-keras backend=tensorflow
+        - py-keras backend=jax
+        - py-keras backend=torch
+        - py-keras-applications
+        - py-keras-preprocessing
+        - py-keras2onnx
+
+        # PyTorch
+        - py-botorch
+        - py-efficientnet-pytorch
+        - py-gpytorch
+        - py-kornia
+        - py-lightning
+        - py-pytorch-gradual-warmup-lr
+        - py-pytorch-lightning
+        - py-segmentation-models-pytorch
+        - py-timm
+        - py-torch
+        - py-torch-cluster
+        - py-torch-geometric
+        - py-torch-nvidia-apex
+        - py-torch-scatter
+        - py-torch-sparse
+        - py-torch-spline-conv
+        - py-torchaudio
+        - py-torchdata
+        - py-torchfile
+        - py-torchgeo
+        - py-torchmetrics
+        - py-torchtext
+        - py-torchvision
+        - py-vector-quantize-pytorch
+
+        # scikit-learn
+        - py-scikit-learn
+        - py-scikit-learn-extra
+
+        # TensorBoard
+        - py-tensorboard
+        - py-tensorboard-data-server
+        - py-tensorboard-plugin-wit
+        - py-tensorboardx
+
+        # TensorFlow
+        - py-tensorflow
+        - py-tensorflow-datasets
+        - py-tensorflow-hub
+        - py-tensorflow-metadata
+        - py-tensorflow-probability
+
+        # XGBoost
+        - py-xgboost
+
+    - variants:
+        - ++mps
+        - ~~mps
+
+  specs:
+    - matrix:
+        - $packages
+        - $variants
```

I think it is:
```suggestion
    - matrix:
        - [$packages]
        - [$variants]
```
here and in other manifest files. Not the best syntax, but that's what we have :slightly_smiling_face: